apps/etterlatte-saksbehandling-ui/client/src/shared/statusbar/Statusbar.tsx

```diff
@@ -18,17 +22,18 @@ export const StatusBar = (props: { theme?: StatusBarTheme; personInfo?: IPersonI
     return GenderList.male
   }
 
+  const navn = `${props.personInfo?.fornavn} ${props.personInfo?.etternavn}`
+
   return (
     <StatusBarWrapper theme={props.theme}>
       {props.personInfo?.fnr && (
         <UserInfo>
           <GenderIcon gender={gender()} />
           <Name>
-            <Link href={`/person/${props.personInfo?.fnr}`}>{props.personInfo?.navn}</Link>{' '}
+            <Link href={`/person/${props.personInfo?.fnr}`}>{navn}</Link>{' '}
           </Name>
           <Skilletegn>|</Skilletegn>
           <Fnr copy value={props.personInfo?.fnr || 'N/A'} />
```

Comment:
Hvorfor sjekkes det om `personInfo` er definert _alle_ steder i denne komponenten? Burde den heller returnert noe default, og så kan alle ? i `personInfo.?` fjernes?

Reply:
Usikker, satte meg ikke så mye in i logikken her 😛